internal/client.go

```diff
@@ -173,10 +178,16 @@ func (client *Client) drawTargetTable() {
 
 func (client *Client) setupUI() {
 	tview.Styles.PrimitiveBackgroundColor = tcell.ColorDefault
-	client.plotTable = tview.NewTable()
-	client.plotTable.SetSelectable(true, false).SetBorder(true).SetTitleAlign(tview.AlignLeft).SetTitle("Active Plots")
-	client.plotTable.SetSelectedStyle(tcell.StyleDefault.Attributes(tcell.AttrReverse))
-	client.plotTable.SetSelectionChangedFunc(client.selectActivePlot).SetFixed(1, 6)
+	client.activePlotsTable = widget.NewSortedTable()
```

Comment:
I find this style easier to read than the "chained dot" style, but `SortedTable` is setup to do either.